jsk_arc2017_common/config/box_sizes.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
I'm not sure what is difference between data and config. What do you think? @pazeshun